packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.css

```
@@ -1,30 +1,20 @@
 lightning-icon {
   --slds-c-icon-color-foreground-default: var(--slds-g-color-brand-base-100, white);
 }
-.slds-visual-picker__figure {
+.visual-picker__figure {
```

Same here for the BEM notation should probably be more something like this:


```suggestion
.caseClassification_visual-picker__figure {
```

---

I know we don't do it everywhere, but it's a recommended practice.

---

`caseClassification__visual-picker__figure`